scripts/api/setup-all.sh

```diff
@@ -27,9 +34,7 @@ done
 command -v jq >/dev/null 2>&1 || { echo >&2 '`jq` ("sed for JSON") is required, but not installed. Download the binary for your platform from http://stedolan.github.io/jq/ and make sure it is in your $PATH (/usr/bin/jq is fine) and executable with `sudo chmod +x /usr/bin/jq`. On Mac, you can install it with `brew install jq` if you use homebrew: http://brew.sh . Aborting.'; exit 1; }
 
 echo "deleting all data from Solr"
-curl http://localhost:8983/solr/collection1/update/json?commit=true -H "Content-type: application/json" -X POST -d "{\"delete\": { \"query\":\"*:*\"}}"
-
-SERVER=http://localhost:8080/api
```

Comment:
Is it strictly necessary to remove the hard coded 8080 in this and other setup scripts?

Reply:
I'm awfully sorry me lad, but yes it is.

When you run this script inside a container which is not the application server itself, `localhost` is that container and not the app server. It's simply like a different machine. If you run this on the host, this works because of the mapped ports, but this will not work on a Windows host. 

Sorry, hardcoding it never was a good idea, same as the paths not being relative to the script location.

Reply:
BTW Jenkins made it to the end, so I'm pretty sure I didn't break the scripts :smile: